Dockerfile

```
@@ -43,4 +34,13 @@ RUN poetry install
 COPY ./tests ./tests
 
 # Define entrypoint 
-ENTRYPOINT ["poetry", "run", "pytest", "-v"]
\ No newline at end of file
+ENTRYPOINT ["poetry", "run", "pytest", "-v"]
+
+#--- PRODUCTION SPECIFIC STEPS ----
+FROM base as production
+
+# poetry setup
+RUN poetry config virtualenvs.create false --local && poetry install
```

You previously explicitly ruled out the dev dependencies with `--no-dev`; we could still do that and keep any unnecessary dependencies out of our production image:
```suggestion
RUN poetry config virtualenvs.create false --local && poetry install --no-dev
```